server/node.go

```diff
@@ -46,6 +47,9 @@ const (
 	gossipGroupLimit = 100
 	// gossipInterval is the interval for gossiping storage-related info.
 	gossipInterval = 1 * time.Minute
+	// runtimeStatsInterval is the interval for logging runtime stats such
+	// as number of goroutines and memory allocated.
+	runtimeStatsInterval = 10 * time.Second
```

Comment:
Actually went back down to 15s. It's nice to have it a little more regular for now.